server/scheduler/GeneticAlgorithm.py

```diff
@@ -225,7 +225,7 @@ def __str__(self): return self._labName
 
 class Schedule:
     # Define a Schedule
-    def __init__(self):
+    def __init__(self, data):
```

Comment:
Why was this change made?

Reply:
@GodwinEkuma Okay. Have you tested to ensure this does not break the algorithm in any way?

Reply:
@Caleb7621-java Only with the GATA data, we would test with Lab and Course as well before merging

Reply:
@GodwinEkuma Let's ensure this works completely before merging then.